packages/dialect-react-ui/components/Chat/provider/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { createContext, FunctionComponent, useContext } from 'react';
+
+// We are duplicating this type in order to have separation of concern: it may be different to what we have in props
+type ChatType = 'inbox' | 'popup' | 'vertical-slider';
+
+interface ChatContextValue {
+  id: string;
```

Comment:
just thinking, shouldn't it be more dialect specific, e.g. `dialectId` to do not confuse with id from html?

Reply:
agree! Great point.